File: src/test/java/application/history/endpoint/history/api/UpdateHistoryTest.java
package application.history.endpoint.history.api;

import application.history.IntegrationTest;
import application.history.WithFaker;
import application.history.history.application.services.UpdateHistoryContentService;
import lombok.SneakyThrows;
import org.junit.jupiter.api.Test;
import org.springframework.beans.factory.annotation.Autowired;


public class UpdateHistoryTest extends IntegrationTest implements WithFaker {

    @Autowired
    private UpdateHistoryContentService service;

    @SneakyThrows
    @Test
    public void getAllHistories() {
    }
}

File: src/history/test/kotlin/com/example/history/application/history/endpoint/history/api/UpdateHistoryTest.kt
package com.example.history.application.history.endpoint.history.api

import application.history.IntegrationTest
import org.junit.jupiter.api.Test

class UpdateHistoryTest : IntegrationTest(), WithFaker {
    @Autowired
    private val service: UpdateHistoryContentService? = null

    @get:Test
    @get:SneakyThrows
    val allHistories: Unit
        get() {}
}
